php-cs-test/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM composer
+
```

Comment:
I've gone ahead and used composer as the official image still as it only needs to collect the file in that step.